server/views/pages/index.njk

```diff
@@ -1,17 +1,13 @@
-{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
-{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
-
 {% extends "../partials/layout.njk" %}
+{% set breadCrumbs = [
+  {
+    text: 'Back to search results',
+    href: '#'
+  }
+] %}
 {% set pageTitle = applicationName + " - Home" %}
 {% set mainClasses = "govuk-body" %}
-{% block beforeContent %}
-  {{ govukBreadcrumbs({
-    items: breadCrumbs 
-  }) }}
-{% endblock %}
 {% block content %}
-  {% if displayBanner === true %}
```

Comment:
Hi @danielburnley, everything looks fine. Just wonder where this logic is moved too? Or do we no longer need it. The photo page does not have a banner.

Reply:
Ah yeah, i moved the photo page so that it extends the layout rather than this index page - which means there's no longer a need to care about the banner displaying/not displaying because that's only on `index.njk`

Reply:
Hi @danielburnley, I just did quick test locally before approving. I get the following error on the photo page.

(/Users/james.grant/Documents/Dev/SSH/hmpps-prisoner-profile/dist/server/views/pages/photoPage.njk) [Line 6, Column 3] parseAggregate: expected comma after expression.

Are you able to see the url ok?
http://localhost:3000/prisoner/G3404UO/image

Reply:
Does the photo page display for you locally?

Reply:
oh - i must have accidentally deleted a comma just before I committed, i've pushed a fix now :D